package.json

```diff
@@ -184,5 +184,6 @@
   "license": "UNLICENSED",
   "msw": {
     "workerDirectory": ".storybook/static"
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
what's this?

Reply:
I got that for free when I ran `yarn` ... by the looks of this issue (https://github.com/nodejs/node/issues/51888) it might be on the way out anyway and isn't necessary, so I'll bin it off ...